react/Breadcrumb.js

```diff
@@ -1,11 +1,12 @@
-import React, { Component } from 'react'
+import React, { Component, Fragment } from 'react'
 import PropTypes from 'prop-types'
-import { injectIntl, intlShape } from 'react-intl'
 import { Link } from 'render'
 
+import HomeIcon from './HomeIcon'
+
 const CSS_CLASSES = {
-  BREADCRUMB: 'vtex-breadcrumb pa4',
-  LINK: 'vtex-breadcrumb__link dib pv1 link light-primary hover-primary ph2',
+  BREADCRUMB: 'vtex-breadcrumb pa4 gray',
+  LINK: 'vtex-breadcrumb__link dib pv1 link ph2 gray hover-near-black',
```

Comment:
Since this object will only contain this property, it's better to move it to render like `linkClassName`

Reply:
👍